samples/portability/cmsis_rtos_v1/philosophers/sample.yaml

```diff
@@ -5,7 +5,7 @@ common:
   tags: cmsis_rtos
   min_ram: 32
   min_flash: 34
-  platform_exclude: qemu_xtensa qemu_x86_64
+  platform_exclude: qemu_xtensa qemu_x86_64 up_squared
```

Comment:
Maybe we should add a comment here about why the exclusion since its a subtle case.

Reply:
Added a comment about this.